mappings/net/minecraft/world/chunk/light/LightStorage.mapping

```diff
@@ -1,4 +1,15 @@
 CLASS net/minecraft/class_3560 net/minecraft/world/chunk/light/LightStorage
+	COMMENT LightStorage handles the access, storage and propagation of a specific kind of light within the world.
+	COMMENT For example, separate instances will be used to store block light as opposed to sky light.
+	COMMENT
+	COMMENT The smallest unit within LightStorage is the section. Sections represent a cube of 16x16x16 blocks and their lighting data.
+	COMMENT In turn, 16 sections stacked on top of each other form a column, which are analogous to the standard 16x256x16 world chunks.
+	COMMENT
+	COMMENT To avoid allocations, LightStorage packs all the coordinate arguments into single long values. Extra care should be taken
```

Comment:
```suggestion
	COMMENT <p>To avoid allocations, LightStorage packs all the coordinate arguments into single long values. Extra care should be taken
```
Without these paragraph split, the rendered javadoc will just have continuous sentences.

Reply:
Thanks :)